thirdparty/CMakeLists.txt

```diff
@@ -51,3 +52,4 @@ if(NOT thrust_POPULATED)
   # We are not using the thrust CMake targets, so no need to call `add_subdirectory()`.
 endif()
 set(THRUST_INCLUDE_DIR "${thrust_SOURCE_DIR}" PARENT_SCOPE)
+set(CUB_INCLUDE_DIR "${thrust_SOURCE_DIR}/dependencies/cub" PARENT_SCOPE)
```

Comment:
For the reasons I stated before, this isn't necessary and shouldn't be done. (Note that in some of the other rapids projects it might make sense to do this -- if they use cub directly, but rmm does not.)

Reply:
See above.